caffe2/serialize/inline_container.cc

```diff
@@ -239,6 +246,8 @@ size_t PyTorchStreamReader::getRecordID(const std::string& name) {
 // return dataptr, size
 std::tuple<at::DataPtr, size_t> PyTorchStreamReader::getRecord(const std::string& name) {
   std::lock_guard<std::mutex> guard(reader_lock_);
+  TORCH_CHECK_VALUE(
```

Comment:
is there any conern of a meaningful perf impact from always checking here? I haven't spent any time with `mz_zip_reader_locate_file` and not sure if `getRecord` is often used with high call volume or with large archives, but looks like this would call it twice per record

Reply:
yeah reading the code more closely, we already have file presence checking in `getRecordID`, it's just the error handling doesn't work. I'll just fix it that way.